api_test.go

```diff
@@ -0,0 +1,300 @@
+// go-api - Client for the Cacophony API server.
+// tests against cacophony-api require apiURL to be pointing
+// to a valid cacophony-api server and test-seed.sql to be run
```

Comment:
the above 2 lines probably belong further down the file, not in the license header